koji/flatten-el8/rhel8-split.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 HOMEDIR=/root/koji-sync/podman/
-BINDIR=/usr/local/bin
+BINDIR=/root/koji-sync/podman/
```

Comment:
The BINDIR appears to be the same as the HOMEDIR now

Reply:
Yes, this is intentional, as the executable is referenced $BINDIR/splitter.py.  (_could_ probably just $HOMEDIR it to minimize redundancy)

Reply:
this should be fixed now.